doc/python/cone-plot.md

```diff
@@ -126,5 +126,10 @@ fig.update_layout(scene=dict(aspectratio=dict(x=1, y=1, z=0.8),
 fig.show()
 ```
 
+### Sizemode
+
+Earlier examples use `sizemode="absolute"` when adjusting the cone size scaling with `sizeref`. `sizemode` also supports `raw`(new in 5.21) and `scaled`. For more details, see the [Cone Sizemode](https://plotly.com/python/reference/cone/#cone-sizemode) and https://plotly.com/python/reference/cone/#cone and [Cone Sizeref](https://plotly.com/python/reference/cone/#cone-sizeref) reference documentation.
+
```

Comment:
Lets's add an example using *raw* sizemode.
Something like the mock used here: https://github.com/plotly/plotly.js/compare/raw-gl3d_cone-wind